infrastructure/sandbox/JITProvisioner/lambda/main.go

```diff
@@ -66,6 +70,44 @@ func applyConfig(c *gin.Context, url, token string) (err error) {
 	return
 }
 
+type MysqlSecretEntry struct {
+    Endpoint string `json:"endpoint"`
+    Username string `json:"username"`
+    Password string `json:"password"`
+}
+
+func clearActivitiesTable(c *gin.Context, id string) (err error) {
+    // Get connection string
+    svc := secretsmanager.New(session.New())
+    sec, err := svc.GetSecretValue(&secretsmanager.GetSecretValueInput{
+        SecretId: aws.String(options.MysqlSecret),
+    })
+    if err != nil {
+        log.Print(err)
+        return
+    }
+    var secretEntry MysqlSecretEntry
+    if err = json.Unmarshal(sec.SecretBinary, &secretEntry); err != nil {
+        log.Print(err)
+        return
+    }
+    connectionString := fmt.Sprintf("%s:%s@tcp(%s)/%s", secretEntry.Username, secretEntry.Password, secretEntry.Endpoint, id)
+    // Connect to db
+    db, err := sql.Open("mysql", connectionString)
+    defer db.Close()
```

Comment:
I think this needs to go below the `if err`, because it can be `nil` when it errors, and then this will panic.